### ISSUE TITLE
Add user as event participants 

### ISSUE THREAD

**Issue by renyidong** [renyidong/PackIt#1]
Currently, even an item is set as shared, since there is only one user, no other user can see the item.

**Comment by LambertYu** [comment]
Yeah, until several users can be grouped into one event, they cannot see the public items in the list. The sharing and inviting people features has not been implemented yet.
